app/routes.js

```diff
@@ -0,0 +1,9 @@
+// const controller = require('./controllers/controller');
+const { healthCheck } = require('./controllers/healthCheck');
+
+exports.init = app => {
+  app.get('/health', healthCheck);
+  // app.get('/endpoint/get/path', [], controller.methodGET);
+  // app.put('/endpoint/put/path', [], controller.methodPUT);
+  // app.post('/endpoint/post/path', [], controller.methodPOST);
+};
```

Comment:
Remove comments, so the kickoff is clean. You can do this in the next PR if you want